src/local/load.go

```diff
@@ -1,17 +1,25 @@
 package local
 
 import (
+	"fmt"
 	"keboola-as-code/src/json"
 	"keboola-as-code/src/manifest"
 	"keboola-as-code/src/model"
 	"keboola-as-code/src/utils"
+	"path/filepath"
 	"reflect"
 )
 
 // LoadModel from manifest and disk
-func LoadModel(projectDir string, record manifest.Record, target interface{}) error {
+func LoadModel(projectDir string, record manifest.Record, target interface{}) (found bool, err error) {
 	errors := &utils.Error{}
 
+	// Check if directory exists
```

Comment:
- Ak je nejaky zaznam v manifeste, tak nasledne sa vola metoda `LoadModel` aby sa nacitali vsetky data k danemu zaznamu (config, row....).
- Pridal som tu túto podmienku a z metody vraciam, ci dany objekt existuje na disku.
- Pouzijem to v `persist` cmd, na detekciu toho, ktore objekty boli vymazane.